pom.xml

```diff
@@ -18,11 +18,23 @@
       <version.slf4j>1.7.32</version.slf4j>
       <version.log4j2>2.17.1</version.log4j2>
     </properties>
+
+    <repositories>
+        <repository>
+            <id>eblocker</id>
+            <url>https://maven.pkg.github.com/P8hJ/eblocker</url>
```

Comment:
This should probably be: https://maven.pkg.github.com/eblocker/eblocker

Reply:
You have eyes like an eagle. Thanks for the tip :-) It is now corrected